### ISSUE TITLE
Some questions about the experiments.

### ISSUE THREAD

**Issue by Z-ZHHH** [ShadeAlsha/LTR-weight-balancing#3]
1. In the code, you use the ResNet34 model for CIFAR100-LT, but in the paper, you use the ResNet32 model.
2. Could you provide more details about the experiment on how to choose the proper weight decay value for long-tailed recognition? It would help a lot.

**Comment by ShadeAlsha** [comment]
1. It is hard to see the difference without seeing your implementation. But can you run my [notebook](https://github.com/ShadeAlsha/LTR-weight-balancing/blob/master/demo1_first-stage-training.ipynb) to see if it reproduces results. One notable difference is that my code uses ResNet34 while you used ResNet32. I apologize for the inconsistent writing and code.
2. We use Bayesian Optimization to search for the weight decay, as explained in Section 3.3. You might find this [implementation](https://github.com/fmfn/BayesianOptimization) useful.
3. While WD tuning atop of MisLAS/BAMLS does not improve much, it seems to suggest the latter methods regularize training in a similar way to WD. Therefore, it is hard to say if WD is (not) the core point of imbalanced learning.

**Comment by rahulvigneswaran** [comment]
Also, the number of epochs is given as 200 in the paper while you have used 320 in the implementation. Can you confirm which is the correct one to replicate the results?

**Comment by ShadeAlsha** [comment]
Yes, we used 200 epochs only for the first stage of training to produce the results in the paper.

**Comment by Z-ZHHH** [comment]
I train ResNet34 with the identical setting, the result is identical to the CoLab experiment. The weight decay 0.005 is not the optimal value for ResNet32 (ResNet32 feature dim=64 while ResNet34 feature dim=512). 
| dataset|Stage |loss |base lr| schedular| batch | epoch| WD | model|result_all|
| --- | ---|---|---|---|---|---|---|---|---|
| CIFAR100-100| stage1 | CE | 0.01 | Coslr | 64 |320|0.005|ResNet34|47.28|
The work is really great.  It would be better and easy to compare with other long-tailed methods if the CoLab experiment is trained with the ResNet32 model.
